json-sports-team/styles.css

```
@@ -0,0 +1,56 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --dark-grey: #0a0a23;
+  --light-grey: #f5f6f7;
+  --white: #ffffff;
+  --black: #000;
+}
+
+body {
+  background-color: var(--dark-grey);
+  text-align: center;
+}
+
+.title,
+.options-label,
+.team-stats {
+  color: var(--white);
+}
```

The `footer` is a nice touch, though the text is hard to read without changing the default color. It might be good to target that element here:

```suggestion
.title,
.options-label,
.team-stats,
footer {
  color: var(--white);
}
```

Or a class would work, too.